src/collectors/windows.plugin/metadata.yaml

```diff
@@ -274,3 +274,150 @@ modules:
               chart_type: line
               dimensions:
                 - name: temperature
+  - meta:
```

Comment:
@thiagoftsm is this PR covering only IIS? If it is supposed to fix the whole metadata YAML, then I see more stuff in the directory that I don't know if they are covered.

PerflibProcesses
PerflibMemory
PerflibThermalzone

Have been implemented.

PerflibMSSQL
PerflibNetwork
PerflibProcessor
PerflibStorage

I see haven't, will you add them in this PR or in a different one?

Reply:
Hello @Ancairon ,

Yes, I am only working on IIS. 

Today, I’ll start working on the remaining Perflibs we’re missing. However, I plan to address each of them in separate PRs to make your review process easier.